_inc/client/plans/single-product-backup.jsx

```diff
@@ -29,56 +30,86 @@ export function SingleProductBackup( { plan, products, upgradeLinks, isFetchingD
 				{ isFetchingData ? (
 					<div className="plans-section__single-product-skeleton is-placeholder" />
 				) : (
-					<SingleProductBackupCard products={ products } upgradeLinks={ upgradeLinks } />
+					<SingleProductBackupCard
+						products={ products }
+						upgradeLinks={ upgradeLinks }
+						// TODO: Wire to picker
+						billingTimeFrame="yearly"
+					/>
 				) }
 			</div>
 		</React.Fragment>
 	);
 }
 
-function SingleProductBackupCard( { products, upgradeLinks } ) {
-	const [ selectedBackupType, setSelectedBackupType ] = useState( 'real-time' );
-	const billingTimeFrame = 'yearly';
+function PromoNudge() {
+	return (
+		<div className="single-product-backup__promo">
+			<div className="single-product-backup__promo-star">
+				<img src={ `${ imagePath }/green-star.svg` } alt="" />
```

Comment:
Since this is a purely decorative image used as a text background, could we define it in the CSS using `background-image` property?

Reply:
We could! I've seen it both ways in the code. What's the best practice here?

Reply:
I guess that the general best practice is not to use `img` for presentational elements if possible.

If we want to keep the `img` tag here, though, let's maybe address @tyxla's suggestion and add the `role="presentation"` here.

Reply:
That makes sense. I've switched to a background image in 8ea1461f4cf65e613c0536b64f0f2a1b3cc5d3a8, which also allowed the markup to be simpler.